content/_data/changelogs/lts.yml

```diff
@@ -7679,6 +7679,225 @@
+      - Langer0416
+    pr_title: Modernise the table design
+    references:
+      - issue: 68912
+      - pull: 6760
+      - pull: 6698
+      - issue: 68934
+      - pull: 6687
+      - issue: 68198
+      - pull: 6772
+    message: |-
+      Update to various UI elements and tables modernized.
+
+  - type: rfe
+    category: rfe
+    pull: 6893
+    authors:
+      - janfaracik
+      - timja
+    pr_title: Add a keyboard shortcut to focus the global search bar (CMD + K/CTRL
+      + K)
+    message: |-
+      Keyboard shortcut added to focus global search bar (CMD + K/CTRL + K).
+
+  - type: rfe
+    category: rfe
+    pull: 6675
+    issue: 68780
+    authors:
+      - Si-So
+      - basil
+    pr_title: "[JENKINS-68780] Blocked upstream projects in the queue block downstream\
+      \ projects "
+    message: |-
+      Blocked upstream projects in the queue block downstream projects when the option "Block build when upstream project is building" is enabled for the downstream project.
+      Similarly, blocked downstream projects in the queue block upstream projects when the option "Block build when downstream project is building" is enabled for the upstream project.
+ 
+  - type: rfe
+    category: rfe
+    pull: 6126
+    authors:
+      - daniel-beck
+      - basil
+      - timja
+    pr_title: Render /manage using ManageJenkinsAction, handle links using StaplerF…
+    message: |-
+      Add breadcrumbs to "Manage Jenkins" and children of it.
+      Developers should ensure they use relative links for navigating between pages if they are a child of "Manage Jenkins".
+
+  - type: rfe
+    category: rfe
+    pull: 6571
+    authors:
+      - basil
+    pr_title: Use `URLClassLoader` by default
+    message: |-
+      Use native Java Platform functionality rather than Ant to load classes.
+      The old behavior can be restored by setting <code>-Dhudson.ClassicPluginStrategy.useAntClassLoader=true</code>.
+
+  - type: rfe
+    category: rfe
+    pull: 6543
+    issue: 51820
+    authors:
+      - jglick
+      - timja
+    pr_title: "[JENKINS-51820] Removing Java Web Start support"
+    references:
+      - pull: 6543
+      - url: https://en.wikipedia.org/wiki/Java_Web_Start
+        title: Java Web Start
+    message: |-
+      Remove Java Web Start support for launching inbound agents, along with the GUI mode, the platform-specific agent installers, and the JAR signature.
+
+  - type: bug
+    category: bug
+    pull: 6692
+    issue: 68390
+    authors:
+      - Hildebrand-Ritense
+      - NotMyFault
+    pr_title: "[JENKINS-68390] Fixed build number cutoff in build history widget"
+    message: |-
+      Fix build number cutoff in build history widget.
+
+  - type: bug
+    category: bug
+    pull: 6651
+    issue: 68672
+    authors:
+      - frankie699
+      - basil
+    pr_title: "[JENKINS-68672] Total length of progress bar still difficult to\
+      \ see in shaded rows"
+    message: |-
+      Make progress bar easier to see.
```

Comment:
Any objections about summing that one up under `Update to various UI elements...`?

To me, that doesn't sound notably enough to get a dedicated changelog entry.

Reply:
Absolutely agree! I'm actually going through now and grouping the smaller UI changes together since there were a few in the backporting list that I didn't have included. I can definitely make sure that this is part of the group though. Thanks so much for the feedback @NotMyFault !!